umm-spec-lib/src/cmr/umm_spec/migration/version/collection.clj

```diff
@@ -238,8 +238,18 @@
       (util/update-in-all [:RelatedUrls :GetService] dissoc :Format)
       doi/migrate-missing-reason-down
       (update-in-each [:PublicationReferences] related-url/migrate-online-resource-down)
-      (update-in-each [:CollectionCitations] related-url/migrate-online-resource-down) 
+      (update-in-each [:CollectionCitations] related-url/migrate-online-resource-down)
       (assoc :UseConstraints (when-let [description (get-in c [:UseConstraints :Description])]
                                ;; Description in 1.10 is object/record.
                                ;; It needs to be converted to string when becoming UseConstraints in 1.9.i
                                (:Description description)))))
+
+(defmethod interface/migrate-umm-version [:collection "1.10" "1.11"]
+  [context c & _]
+  (-> c
+    (related-url/migrate-up-to-1_11)))
```

Comment:
alignment is off for this, needs to be vertically aligned.

Reply:
Probably don't need to use a thread here anyway.

Reply:
FYI, we try to use https://github.com/bbatsov/clojure-style-guide

Reply:
done